packages/schematics/integration/integration.spec.ts

```diff
@@ -0,0 +1,260 @@
+            'CREATE projects/second-app-e2e/src/plugins/index.ts .*',
+            'CREATE projects/second-app-e2e/src/support/commands.ts .*',
+            'CREATE projects/second-app-e2e/src/support/index.ts .*',
+            'UPDATE package.json .*',
+            'UPDATE angular.json .*',
+            'UPDATE projects/second-app-e2e/tsconfig.e2e.json .*',
+            '> cypress@.* postinstall .*',
+            'added .* packages .*'
+          )
+        );
+      });
+
+      it('should setup e2e runner for cypress', async () => {
+        await testBed.execute('ng generate app second-app');
+        await linkSchematics(testBed);
+        await testBed.execute(
+          'ng generate @co-it/schematics:cypress --app=second-app --overwrite=true --no-interactive'
+        );
+
+        const result = await testBed.run('ng e2e second-app-e2e --headless');
+
+        expect(result.stdout).toMatch(
+          matchLines('examples/app.spec.js', 'All specs passed!')
+        );
+      });
+    });
+  });
+
+  describe('@co-it/schematics:jest', () => {
+    describe('When "ng generate @co-it/schematics:jest --hook=true" is run', () => {
+      it('should update files', async () => {
+        const result = await testBed.run(
+          'ng generate @co-it/schematics:jest --hook=true'
+        );
+
+        expect(result.stdout).toMatch(
+          matchLines(
+            'DELETE src/karma.conf.js',
+            'DELETE src/test.ts',
+            'CREATE jest.config.js',
+            'CREATE src/setup-jest.ts',
+            'CREATE src/test-config.helper.ts',
+            'CREATE .huskyrc',
+            'UPDATE package.json',
+            'UPDATE src/tsconfig.spec.json',
+            '> husky@.* install',
+            'added .* packages'
+          )
+        );
+      });
+
+      it('should run jest tests', async () => {
+        await testBed.execute('ng generate @co-it/schematics:jest --hook=true');
+
+        const result = await testBed.run('yarn test');
+
+        // Jest uses stderr, see issue https://github.com/facebook/jest/issues/5064
+        expect(result.stderr).toMatch(
+          matchLines(
+            'PASS src/app/app.component.spec.ts',
+            'AppComponent',
+            ' should create the app',
+            " should have as title 'integration-test'",
+            ' should render title in a h1 tag',
+            'Test Suites: 1 passed, 1 total',
+            'Tests:       3 passed, 3 total',
+            'Snapshots:   0 total',
+            'Time: .*',
+            'Ran all test suites.'
+          )
+        );
+      });
+    });
+  });
+
+  describe('@co-it/schematics:prettier', () => {
+    describe('When "ng generate @co-it/schematics:prettier --hook=true" is run', () => {
+      it('should update files', async () => {
+        const result = await testBed.run(
+          'ng generate @co-it/schematics:prettier --hook=true'
+        );
+
+        expect(result.stdout).toMatch(
+          matchLines(
```

Comment:
**Idea** We can make a custom matcher out of `matchLines`.

Reply:
Yeah, definitely a good idea! But I would propose doing it after current version is in master. What do you think?

Reply:
Of cause, we really need this PR merged before going on with new stuff.